tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -4657,6 +4690,61 @@ def _assert_is_fit(self, message_suffix: str = None):
                 error_message = f"{error_message} `.{message_suffix}`."
             raise AssertionError(error_message)
 
+    @Deprecated(min_version_to_warn="0.8", min_version_to_error="1.1", version_to_remove="1.1", new="persist")
+    def persist_models(self, **kwargs) -> List[str]:
+        """Deprecated method. Use `persist` instead."""
+        return self.persist(**kwargs)
+
+    @Deprecated(min_version_to_warn="0.8", min_version_to_error="1.1", version_to_remove="1.1", new="unpersist")
+    def unpersist_models(self, **kwargs) -> List[str]:
```

Comment:
Maybe change all deprecated method signatures to `def method(self, *args, **kwargs):` to avoid breaking calls that use positional arguments?

Reply:
Added `*args`, good catch!